README.rst

```diff
@@ -48,48 +48,7 @@ Run a development Cassandra node:
 
 .. code-block:: bash
 
-    $ docker run --name dev-cassandra -d -p 9042:9042 cassandra:2.1
-
-Insert schema (you might need to wait a few seconds for Cassandra to boot):
-
-.. code-block:: bash
-
-    $ docker run -i --link dev-cassandra:cassandra --rm cassandra:2.1 cqlsh cassandra < schema.cql
-
-General cqlsh access to your dev instance:
```

Comment:
I think we should leave this "general cqlsh access" part in the README (after ./setup-dev-cassandra.sh) so people unfamiliar with Docker/Cassandra now how to look into it without digging into the shell script :smile: